README.md

```diff
@@ -439,3 +439,30 @@ We recommend querying using payment key hashes (`addr_vkh`) when possible (other
 
 
 </details>
+
+<details>
+  <summary>milkomeda</summary>
+
+  This endpoint is used to get milkomeda info data (minValue, address & tokens list per deployment/network)
+
+  Input
+
+  None (GET request)
+
+  Output
+
+  ```js
+  {
+    minimumValue: number,
+    address: string,
+    tokensPerNetwork: { // proposed networks are the most probable for now
+      internalTestnet: {
+        tokens: [ { name: string, policyId: string }, ]
+      },
+      publicMainnet: {
```

Comment:
Do the names "publicMainnet" and "internalTestnet" here come from anything? I don't think we should have to use these at all (see above comment), but if we do probably these strings should come from the NPM package network names instead of being hard-coded here

Reply:
Yes, they don't have genesis - I added them, because I didn't know if we should specify such stuff already or relay on something later. I'll improve that.